spec/spec_helper.cr

```diff
@@ -1,5 +1,9 @@
-# NOTE: Constants should be set before require begins.
+if ENV["TEST_SUITE"] == "./spec/amber"
+  puts "WAITING: sleep 1.minute before running the amber specs to ensure all is ok..."
+  sleep 1.minute
```

Comment:
sleeping a whole minute feels like overkill. Could we do something more intelligent, like try to connect with the database and sleep 100ms when it fails to connect?

Reply:
> sleeping a whole minute feels like overkill.

I was inspired by TechEmpower Benchmark, TFB used to wait a whole minute before running test, I guess they removed it after replacing all by docker :sweat_smile: 

>  like try to connect with the database and sleep 100ms when it fails to connect?

Oh, ok, what command do you suggest? `psql`, `amber exec`?

Reply:
Travis is still failing I guess this can do the work:

^ https://travis-ci.org/amberframework/amber/jobs/363480820#L1454

Maybe I can wait less time, `30.seconds`, WDYT? :sweat_smile:

Reply:
https://travis-ci.org/amberframework/amber/jobs/363482182#L1454

^ Failing here as well :sweat_smile:

Reply:
What if we add this to he build and shell out to it before starting the tests? https://github.com/vishnubob/wait-for-it

Reply:
Maybe I can create a loop to check if connection and tables are created, like you suggested, WDYT?

I think adding a new dependence to the spec suit is an non required overload :sweat_smile: